pynamodb/indexes.py

```diff
@@ -151,27 +152,27 @@ def _get_schema(cls) -> Dict:
         """
         Returns the schema for this index
         """
-        attr_definitions = []
-        schema = []
+        schema = {
+            'index_name': cls.Meta.index_name,
+            'key_schema': [],
+            'projection': {
+                PROJECTION_TYPE: cls.Meta.projection.projection_type,
+            },
+        }
```

Comment:
This now matches the index schema for the dynamodb create table api:
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_CreateTable.html#DDB-CreateTable-request-LocalSecondaryIndexes